doc/contributing/releases.md

```diff
@@ -1069,53 +1069,7 @@ have the right internal version strings. Check that the API docs are available
 at <https://nodejs.org/api/>. Check that the release catalog files are correct
 at <https://nodejs.org/dist/index.tab> and <https://nodejs.org/dist/index.json>.
 
-### 17. Create a blog post
-
-There is an automatic build that is kicked off when you promote new builds, so
-within a few minutes nodejs.org will be listing your new version as the latest
-release. However, the blog post is not yet fully automatic.
-
-Create a new blog post by running the [nodejs.org release-post.js script][]:
-
-```bash
-node ./apps/site/scripts/release-post/index.mjs x.y.z
-```
-
-This script will use the promoted builds and changelog to generate the post. Run
-`npm run serve` to preview the post locally before pushing to the
-[nodejs.org repository][].
-
-* You can add a short blurb just under the main heading if you want to say
-  something important, otherwise the text should be publication ready.
-
-* The links to the download files won't be complete unless you waited for the
-  ARMv6 builds. Any downloads that are missing will have `*Coming soon*` next to
-  them. It's your responsibility to manually update these later when you have
-  the outstanding builds.
-
-* The `SHASUMS256.txt.asc` content is at the bottom of the post. When you update
-  the list of tarballs you'll need to copy/paste the new contents of this file
-  to reflect those changes.
```

Comment:
This should not be removed. In general, it's probably a good idea to keep the old way documented and only mention the workflow will open the PR